docs/docs/dev_docs/contracts/portals/data_structures.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Data Structures
+---
+
+The `DataStructures` are structs that we are using throughout the message infrastructure and registry.
+
+**Links**: [Implementation](https://github.com/AztecProtocol/aztec-packages/blob/master/l1-contracts/src/core/libraries/DataStructures.sol).
+
+## `Entry`
+
+An entry for the messageboxes multi-sets. 
```

Comment:
```suggestion
An entry for the message box (the entry acts like a multi-set, enabling multiple entries for the same message)
```

Reply:
The entry does not act like a multiset, it is an entry in a multisit.